src/utils/__tests__/index.ts

```diff
@@ -1962,7 +1962,7 @@ describe('createClaim', () => {
   test('', () => {
```

Comment:
Can you add a description for this test?

Reply:
Done